plaid/client.py

```diff
@@ -61,7 +61,7 @@ def store_access_token(func):
     def inner_func(self, *args, **kwargs):
         response = func(self, *args, **kwargs)
         if response.ok:
-            json_data = json.loads(response.content)
+            json_data = json.loads(response.content.decode())
```

Comment:
@erikbern, so with Python 3, json loads fails with binary encoding?

I think that it would be nice to refactor the decoding logic into a json loads utility helper function.